ar_track_alvar/test/test_launch_files.py

```diff
@@ -0,0 +1,85 @@
+from rclpy.duration import Duration
```

Comment:
All of the launch files actually spin up and run?

Reply:
I believe so, the test passes.